components/org.wso2.carbon.dashboards.portal/src/main/fragments/bar-chart/widgetConf.json

```diff
@@ -0,0 +1,38 @@
+{
+  "info": {
+    "id": "bar-chart",
+    "displayName": "Bar Chart",
+    "thumbnail": "widget-thumbnail.jpg"
+  },
+  "providerConfig": {
+    "stream": "",
+    "version": "",
+    "interval": 500,
+    "type": "",
+    "metadata": {
+      "names": [
+      ],
+      "types": [
+      ]
+    }
+  },
+  "chartConfig": {
+    "type": "",
+    "x": "",
+    "charts": [
+      {
+        "y": ""
+      }
+    ],
+    "maxLength": 20,
+    "width": 800,
+    "height": 400
+  },
+  "pubsub": {
+    "isPublisher": true,
+    "isSubscriber": false,
+    "topic": ""
+  },
+  "permission": {
```

Comment:
No. According to widget specification, it should be as above. Please refer https://docs.google.com/document/d/1QSdUvbnlnytuO_29Sukrg_I0P4-bXkBPFS6M-X6rI80/edit